packages/components/psammead-brand/src/index.jsx

```diff
@@ -164,6 +172,30 @@ StyledBrand.defaultProps = {
   serviceLocalisedName: null,
 };
 
+const ScriptLinkWrapper = styled.div`
+  display: inline-block;
+  @media (max-width: ${GEL_GROUP_0_SCREEN_WIDTH_MAX}) {
+    padding-top: 0;
+    position: relative;
+    right: 0;
+    display: block;
+  }
+
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    top: calc(50% - 1.25rem);
+  }
+  position: absolute;
+  ${({ dir }) => (dir === 'ltr' ? 'right: 0' : 'left: 0')};
+  top: calc(50% - 1.5rem);
```

Comment:
The `top` calculation is wrong when the screen is small, making the link to go outside the banner when it should stay in it.

<img width="258" alt="brand-wrong" src="https://user-images.githubusercontent.com/46446236/67426613-5d5faf80-f5d2-11e9-81c8-a91f8faf1fd8.png">
 ⬇
<img width="265" alt="brand-right" src="https://user-images.githubusercontent.com/46446236/67426618-5f297300-f5d2-11e9-98f4-6fc5d63fe07e.png">

Instead of making it absolute and calculate the top position, an alternative would be to add a new `div` to wrap both, Brand and Script Link, and use flex to align the items vertically and put them on each side. And then you wouldn't need `ScriptLinkWrapper` and `SvgWrapper`, just to apply a couple of tweaks to the `Banner` height and padding. Up to you :)

```
const StyledWrapper = styled.div`
  display: flex;
  justify-content: space-between;
  align-items: center;
  flex-wrap: wrap;
  max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
  margin: 0 auto;
`;
```

```
  <Banner
      svgHeight={svgHeight}
      borderTop={borderTop}
      borderBottom={borderBottom}
      backgroundColour={backgroundColour}
      logoColour={logoColour}
      {...rest}
    >
      <StyledWrapper>
        {url ? (
          <StyledLink href={url} maxWidth={maxWidth} minWidth={minWidth}>
            <StyledBrand {...props} />
          </StyledLink>
        ) : (
          <StyledBrand {...props} />
        )}
        {scriptLinkComponent}
      </StyledWrapper>
    </Banner>

```

Reply:
@DenisHdz Good point. Done 😎